QualityControl/lib/services/ccdb/CcdbService.js

```diff
@@ -176,7 +176,7 @@ export class CcdbService {
    * ```
    * @param {CcdbObjectIdentification} partialIdentification - fields such as path, validFrom, etc.
    * @returns {Promise.<CcdbObjectIdentification>} - returns object full identification
-   * @throws {Error}
+   * @throws {Error} throws if the object cannot be found
```

Comment:
Perhaps it should be `NotFoundError` when the object is not found? More specifically when CCDB returns 0 objects